client/lib/sites-list/list.js

```diff
@@ -72,18 +72,24 @@ SitesList.prototype.get = function() {
  * @api public
  */
 SitesList.prototype.fetch = function() {
-	if ( this.fetching ) {
+	var userIsLoggedIn = Boolean( user.data );
```

Comment:
I wonder if we should add a `isLoggedIn()` function to the `User` class rather than relying on this sort of check. This would reduce code duplication and make sure all our code follow the same pattern when it comes to checking if a user is logged in or not.